packages/vite/src/node/plugins/html.ts

```diff
@@ -367,7 +367,7 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
               prop.name === 'style' &&
               prop.type === NodeTypes.ATTRIBUTE &&
               prop.value &&
-              prop.value.content.includes('url(') // only url(...) in css need to emit file
+              /url\s*\(/.test(prop.value.content) // only url(...) in css need to emit file
```

Comment:
Change it to `url(*)` so that it will not it will not avoid any characters in the middle

Reply:
may be `url(` is ok? 
I think the middle can be any characters?

Reply:
that would grab all characters until the targeted string ends.
For example:
```
it would select:
url(my-god-damn-png-file.png); and some other css properties too 
```
This would overflow the test, so you should stick on to using `url(*)`

Reply:
thanks

Reply:
![image](https://user-images.githubusercontent.com/36070057/151187473-6ae0bb28-1780-4f9c-ae3b-cb2bd4d65188.png) it also grab the `url(` will break the crap?

Reply:
@poyoho , What I meant was we only want to emit the file if there is a complete syntax of url inside the css. 
Something like this:
<img width="538" alt="Screenshot 2022-01-26 at 8 38 13 PM" src="https://user-images.githubusercontent.com/72091386/151189314-b8493da6-8e85-4361-b152-254f91626cc3.png">

Reply:
I know your mean now! 🙈